packages/augur-ui/src/utils/can-claim-proceeds.ts

```diff
@@ -1,7 +1,3 @@
-import { createBigNumber } from "utils/create-big-number";
-import { CONTRACT_INTERVAL } from "modules/common/constants";
-
-// Check that the CLAIM_PROCEEDS_WAIT_TIME has passed since finalization
 export default function canClaimProceeds(
```

Comment:
I think this method can be deleted. we should get the same information from positions unClaimed... being greater than 0.